### ISSUE TITLE
Focus History Combo Box not retaining focus runs following a N.I.N.A shutdown/retsart

### ISSUE THREAD

**Issue by CheekyMonkey** [ghilios/joko.nina.plugins#14]
N.I.N.A. 3.0  Nightly #54
Hocus Focus 3.0.0.12
A couple of weeks back, I lost power to my astro computer running N.I.N.A. during an imaging run.

Since then the dockable focus curves view in N.I.N.A. no longer retains the full history of focus runs between imaging sessions
I do however continue to see **all** Hocus Focus focus runs in the Hocus Focus folder on my astro pc

During a given imaging session, that session's Hocus Focus runs **do appear** in the focus history combo box (see screen capture below)

Once I then shut down and then restart N.I.N.A., the combo box comes up as empty

![image](https://github.com/ghilios/joko.nina.plugins/assets/537672/cc050b43-3ed0-425c-8a2c-5373b00d9934)

**Comment by CheekyMonkey** [comment]
Update:
I ran an imaging session last night with N.I.N.A.

Before I started I removed all files from %LOCALAPPDATA%\NINA\AutoFocus\ and also removed all files from my HF save directory (D:\NINA\_HocusFocus\) so both folders were empty

A number of successful HF AF runs took place during the session last night
I can see that all AF .json files and all HF AF files are present in their respective folders

I shut down N.I.N.A. this morning and restarted N.I.N.A.
I observe that the HF/AF combobox in the anchorable view is empty. 
Looks like, for some reason, the combobox is still not able to pick up on the HF AF history following N.I.N.A. shutdown/restart

![image](https://github.com/ghilios/joko.nina.plugins/assets/537672/ed1b5cbb-14a5-4197-82ec-2565ccb943fc)

![image](https://github.com/ghilios/joko.nina.plugins/assets/537672/86c7e027-3d08-48f6-95f7-b7e0fca92d0b)

![image](https://github.com/ghilios/joko.nina.plugins/assets/537672/1f39ef9e-4501-4c35-894c-f876fb2a4de4)
